lib/ansible/modules/packaging/os/swupd.py

```diff
@@ -197,10 +197,6 @@ def install_bundle(self, bundle):
             self.msg = "Bundle %s installed" % bundle
             return
 
-        if self.rc == 18:
-            self.msg = "Bundle name %s is invalid" % bundle
```

Comment:
Was this message less accurate than the one being set now ("Failed to install bundle %s" % bundle) if self.rc == 18?

Reply:
Seems helpful to know if the bundle name is wrong

Reply:
Thanks @s-hertel for taking a look at this. I did think about this and my motivation to just remove the entire special error case check was:

1. The error message coming from swupd already tells about all the bundles with bad names. The message from swupd is clear.
2. I thought relying on the special return code 18 is a bit prone to errors. Just propagating whatever swupd tells is simpler and more robust.